src/components/Sidebar/SearchResults.test.js

```diff
@@ -21,7 +21,29 @@ describe('SearchResults', () => {
       term: '4720',
       type: 'course',
     }];
-    const wrapper = shallow(<SearchResults searchResults={testResults} />);
+    const wrapper = shallow(<SearchResults searchResults={testResults} isFetching={false} />);
+
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('renders loading correctly', () => {
+    const testResults = [{
```

Comment:
Since this is identical to the testResults that we use for the first spec, we can move this const variable to be declared within the scope of the suite instead of each spec (in other words, put this declaration on line 6). That'll make it available to use in all of the following specs.